core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -54,9 +55,9 @@ private[spark] class SparkHadoopUtil extends Logging {
    * Runs the given function with a Hadoop UserGroupInformation as a thread local variable
    * (distributed to child threads), used for authenticating HDFS and YARN calls.
    *
-   * IMPORTANT NOTE: If this function is going to be called repeated in the same process
-   * you need to look https://issues.apache.org/jira/browse/HDFS-3545 and possibly
-   * do a FileSystem.closeAllForUGI in order to avoid leaking Filesystems
+   * IMPORTANT NOTE: If this function is going to be called repeated in the same process you need
```

Comment:
Is there any change in the comment content? If not, please don't make unnecessary format modifications.

Reply:
For comments only?

Reply:
Rolled back unnecessary formatting